liblwgeom/cunit/cu_triangulate.c

```diff
@@ -13,8 +14,7 @@
 #include "CUnit/Basic.h"
 #include "cu_tester.h"
 
-#include "liblwgeom.h"
-#include "liblwgeom_internal.h"
+#include "../liblwgeom_internal.h"
```

Comment:
Most of the test files include liblwgeom_internal.h instead of liblwgeom.h.  This one had both for some reason (redundant since liblwgeom_internal.h includes liblwgeom.h), so I removed one to match the general convention of the others.